pkg/autoupgrade/daemon.go

```diff
@@ -210,9 +186,9 @@ func (d *daemon) sync(ctx context.Context) error {
 		var tags []string
 		var pullErr error
 		if hasValidRegistry {
-			_, tags, pullErr = images.ListTags(ctx, d.client, imageKey.namespace, imageKey.image)
+			tags, pullErr = d.client.listTags(ctx, imageKey.namespace, imageKey.image)
 		}
-		localTags, err := tags2.GetTagsMatchingRepository(current, ctx, d.client, "acorn", defaultNoReg)
+		localTags, err := d.client.getTagsMatchingRepo(ctx, current, "acorn", defaultNoReg)
```

Comment:
Should `"acorn"` here be `imageKey.namespace` instead?

Reply:
@cjellick Before we merge, I just want to verify whether this is correct. With all the "project" work, maybe this should be `imageKey.namespace` (which is the `app.Namespace`)?

Reply:
EDIT: This isn't relevant with the recent changes, and, based on my investigation, this should be `imageKey.namespace` in the "project" paradigm.